fastlane_bot/events/utils.py

```diff
@@ -627,6 +628,13 @@ def get_config(
         cfg.w3_async = AsyncWeb3(AsyncWeb3.AsyncHTTPProvider(rpc_url))
         if 'tenderly' in rpc_url:
             cfg.NETWORK = cfg.NETWORK_TENDERLY
+        cfg.WEB3_ALCHEMY_PROJECT_ID = rpc_url.split("/")[-1]
```

Comment:
This code is OS-dependent.
Specifically, it will not work on Windows OS, I believe.
Consider using `os.path.basename` (or something similar) instead.

Reply:
this is a URL, not a path, which as far as I know these are not OS specific

Reply:
My bad; didn't notice that.